test/integration/image-optimizer/test/util.ts

```diff
@@ -974,15 +996,21 @@ export function runTests(ctx: RunTestsCtx) {
         expect(five.res.headers.get('Content-Disposition')).toBe(
           `${contentDispositionType}; filename="slow.webp"`
         )
-        await check(async () => {
+        if (!ctx.isrFlushToDisk) {
```

Comment:
Let's update all these checks in `util.ts` to be more explicit:

```
if (ctx.isrFlushToDisk === false)
```

That way we don't consider `undefined` and `false` the same because we know the default behavior of `undefined` is actually `true`.